lpc845/README.md

```diff
@@ -0,0 +1,78 @@
+# LPC845 PAC [![crates.io](https://img.shields.io/crates/v/lpc845-pac.svg)](https://crates.io/crates/lpc845-pac) [![Documentation](https://docs.rs/lpc845-pac/badge.svg)](https://docs.rs/lpc845-pac) [![Build Status](https://travis-ci.com/lpc-rs/lpc-pac.svg?branch=master)](https://travis-ci.com/lpc-rs/lpc-pac)
+
+## Introduction
+
+**This is a very low-level library. Most users should use [lpc82x-hal] instead.**
+
+Low-level register mappings for the [NXP LPC845] family of ARM Cortex-M0+ microcontrollers, written in [Rust]. The code is generated automatically from the [SVD file] available from ARM, using [svd2rust].
+
+The purpose of this crate is to give embedded programs or libraries written Rust access to the complete functionality of LPC845 MCUs.
+
+Please also check out [lpc82x-hal], a higher-level crate for interfacing with LPC82x microcontrollers.
+
+
+## Usage
+
+Add this to the `[dependencies]` section of your `Cargo.toml` to include rust-lpc82x in your Cargo project:
+
+``` toml
+lpc845 = "0.6"
```

Comment:
Name of the crate is `lpc845-pac`, or maybe it's going to be `lpc84x-pac`. Either way, this is wrong :-)

Version is wrong, too.